src/assessment.js

```diff
@@ -49,5 +49,10 @@ var q2 = [/* your answer here */];
 // executing it.
 //
 var q5 = function fireOnAlderaan () {
-  // your code here
+  alderaan.call(fire);
```

Comment:
No, sorry. `alderaan` doesn't have a `call` method, only functions do. You've got the pieces, but their not put together in the right order.